src/Operators/remapping.jl

```diff
@@ -108,6 +110,35 @@ function overlap_weights(
     return kron(Y_ov, X_ov)
 end
 
+function overlap_weights(
+    target::T,
+    source::S,
+) where {
+    T <: SpectralElementSpace1D{<:IntervalTopology, Quadratures.GL{1}},
+    S <: SpectralElementSpace1D{<:IntervalTopology, Quadratures.GL{1}},
+}
```

Comment:
Just my little OCD speaking... what do you think about moving the 1D case up before the 2D case?